### PR TITLE
Convert to use puppet_metadata

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by Flipez** [comment]
Hi @ekohl , is this stil WIP?

**Comment by ekohl** [comment]
I still need to do a gem release of puppet_metadata but due to a vacation I haven't gotten around to that yet.

**Comment by Flipez** [comment]
No worries ;)

**Comment by ekohl** [comment]
Rebased and changed the references to the released version. I think it's ready to review now. I don't have a test env for the project itself and it looks like CI itself on the project is broken now. I'm currently blocked on that.

File: Gemfile [hunk 1]
```diff
@@ -39,6 +39,8 @@ gem 'redis'
 
 gem 'puppet_forge'
 
+gem 'puppet_metadata', '~> 0.1'
```
Comment (ekohl) [inline Gemfile:42]: Want to bump to 1.0 too?

**Comment by Flipez** [comment]
Checks are looking good again now:

grafik

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**